### PR TITLE
Use friendly drive names

### PR DESCRIPTION
**Pull request by NZTimKeegan**
Adds volume name to drive names to make it easier to select a target

### CODE REVIEW

**Comment by NZTimKeegan** [comment]
This has been created to solve issue #28. Here are some examples of how this looks.

Adding a drive:
Tardis drive select example

Backing up:
Tardis backing up example

Out of date backups:

File: src/Utils.vala [hunk 1]
```diff
@@ -81,4 +81,11 @@ public class Tardis.Utils {
 
         return new_list;
     }
+
+    public static string display_name (GLib.Volume volume) {
+        var volume_name = volume.get_name ();
+        var drive_name = volume.get_drive ().get_name ();
```
Comment (chasinglogic) [inline src/Utils.vala]: `get_drive` returns a `Drive?` so this should probably handle the null case.

Either set a reasonable default or just return `volume_name` if it's null.

Comment (NZTimKeegan) [reply src/Utils.vala]: Fixed but I've squashed it into the wrong commit sorry - one moment and I'll sort it out

Comment (NZTimKeegan) [reply src/Utils.vala]: OK I think I've got it

**Pull request merged**